inc/spbc-settings.php

```diff
@@ -1455,22 +1460,26 @@ function spbct_settings__the_settings_tab_draw($elem_name, $elem)
  */
 function spbc_settings__get_quick_navbar_html($elem)
 {
+    global $spbc;
+
     $nav_link_template = '<a href="#%s" id="spbc-quicknav--link-%s"><span class="spbc_manual_link spbc-quicknav--link"><i class="spbc-icon-left-dir"></i>%s</span></a>';
     $nav_bar_template = '
         <div id="spbc-quicknav--links_wrapper" class="spbc-quicknav--links_wrapper">
         <h4 id="spbc_quicknav--header">%s</h4>
         %s
         <input type="submit" name="submit" id="submit" class="spbc_quicknav--save_button button-primary" value="Save Changes">
         </div>';
-
     $nav_elements = isset($elem['sections']) && is_array($elem['sections']) ? array_keys($elem['sections']) : array();
     $nav_links_html = '';
-
     foreach ($nav_elements as $_index => $nav_element) {
         //skip banner section from links list
         if ($nav_element === 'section_top_banner') {
             continue;
         }
+        //Skip the rendering of the menu block if the White Label mod is enabled
+        if ($nav_element === 'spbc_trusted_and_affiliate' && $spbc->data["wl_company_name"]) {
```

Comment:
`$spbc->data["wl_company_name"]` is always defined. Please, check WL enabled by `$spbc->data["wl_mode_enabled"]`